### DISCUSSION TITLE
Stop saying "Chrome" Extension

### DISCUSSION THREAD

**Discussion by takuyahara** [wxt-dev/wxt#1201] [Ideas]
As a person who didn't know about making extension at all, I like wxt that allowed me to make it in a day, cheers for all who made and maintains. However I'd like to propose not to say "Chrome" Extension in many pages, particularly in top page because it misleads that wxt is only for Chrome. It's written that wxt is not only for Chrome just below but if pages are seen quickly by those who are comparing wxt with other competitors for instance, you may lose a chance to attract them. Hope this helps.

**Comment by aklinker1** [comment]
Main reason it's used is because that's the SEO term people search for... But you're right, I don't particularly care if we're high on SEO, so we can change that wording. Can you open A PR with the ones you noticed and I can change that others you missed?

**Reply by takuyahara** [reply]
Done, please refer to https://github.com/wxt-dev/wxt/pull/1229.
